src/client/api/restful/get_outputs_id.c

```diff
@@ -258,14 +271,14 @@ int get_outputs_from_nft_id(iota_client_conf_t const *conf, char const nft_id[],
   }
 
   size_t id_len = strlen(nft_id);
-  if (id_len != ADDRESS_NFT_HEX_BYTES) {
+  if (id_len != BIN_TO_HEX_BYTES(NFT_ID_BYTES)) {
     printf("[%s:%d] incorrect length of id\n", __func__, __LINE__);
     return -1;
   }
 
   // compose restful api command
-  char cmd_buffer[67] = {0};  // 67 = max size of api path(28) + ADDRESS_NFT_HEX_BYTES(40) + 1
-  int snprintf_ret = snprintf(cmd_buffer, sizeof(cmd_buffer), "/api/plugins/indexer/v1/nft/%s", nft_id);
+  char cmd_buffer[70] = {0};  // 70 = max size of api path(29) + BIN_TO_HEX_BYTES(NFT_ID_BYTES)(40) + 1
```

Comment:
What `+ 1` represents in a comment? NULL string terminator? If so, then `BIN_TO_HEX_STR_BYTES(NFT_ID_BYTES)` could be used.

Reply:
For (id_len != BIN_TO_HEX_BYTES(ALIAS_ID_BYTES)), we cannot use BIN_TO_HEX_STR_BYTES(NFT_ID_BYTES).

Reply:
Yes this is true.
I just mean to change comment like this:
```suggestion
  char cmd_buffer[70] = {0};  // 70 = max size of api path(29) + BIN_TO_HEX_STR_BYTES(NFT_ID_BYTES)(41)
```

Reply:
But actually, the null character is not for BIN_TO_HEX_STR_BYTES(NFT_ID_BYTES). It is for the cmd_buffer to have null character.

Reply:
Ok. Then just ignore my suggestion.